docs/en/connector-v2/EnvConf.md

```diff
@@ -0,0 +1,19 @@
+# EnvConf
+
+This document describes env configuration information,env unifies the environment variables of all engines.
+
+## jars
```

Comment:
Add `jars` to file :https://github.com/apache/incubator-seatunnel/pull/3353/files#diff-6f68c25dd21780017dc98d220278af32f5ced954e0a0cd4ef3f585fd3f85380e ?